### PR TITLE
fix: verify connection outside the transaction

### PR DESCRIPTION
**Pull request by yordis**

### CODE REVIEW

File: lib/sequin_web/live/databases/form.ex [hunk 1]
```diff
@@ -332,23 +333,37 @@ defmodule SequinWeb.DatabasesLive.Form do
...
+        with {:ok, db} <- res do
           # It's now safe to start the replication slot
           ReplicationSupervisor.start_replication(db.replication_slot)
-          # Safe to update health here because we just validated that the database is reachable
-          # TODO: Implement background health updates for reachability
-          Health.update(db, :reachable, :healthy)
+
+          %{postgres_database_id: db.id}
+          |> PostgresDatabaseHealthChecker.new_from()
+          |> Oban.insert()
```
Comment (acco) [inline lib/sequin_web/live/databases/form.ex:342]: Perform a check right away, I like that 👍

**Review by acco — APPROVED** [review]
Looks good! Just stack another commit or PR on top of this one that adds the "Test connection" button to the form, then when that's in we can merge!

File: lib/sequin/health_runtime/postgres_database_health_checker.ex [hunk 2]
```diff
@@ -1,35 +1,29 @@
-defmodule Sequin.Health.PostgresDatabaseHealthChecker do
+defmodule Sequin.HealthRuntime.PostgresDatabaseHealthChecker do
   @moduledoc false
-  use GenServer
+
+  use Oban.Worker,
+    queue: :health_checks,
+    max_attempts: 1,
+    unique: [states: [:available, :scheduled, :executing, :retryable]]
```
Comment (yordis) [inline lib/sequin/health_runtime/postgres_database_health_checker.ex:7]: @acco could you verify here please?

File: assets/svelte/databases/Form.svelte [hunk 3]
```diff
@@ -1,11 +1,12 @@
 <script lang="ts">
   import { onMount } from "svelte";
-  import { slide } from "svelte/transition";
+  import { slide, fade } from "svelte/transition";
```
Comment (acco) [inline assets/svelte/databases/Form.svelte:3]: fancy

**Review by acco — APPROVED** [review]
Great work!

**Pull request merged**